docs/data/charts/zoom-and-pan/ZoomSliderPreview.tsx.preview

```diff
@@ -0,0 +1,16 @@
+<ToggleButtonGroup
+  value={chartType}
+  exclusive
+  onChange={handleChartType}
+  aria-label="chart type"
+  fullWidth
+>
+  {['bar', 'line', 'scatter'].map((type) => (
+    <ToggleButton key={type} value={type} aria-label="left aligned">
+      {type}
+    </ToggleButton>
+  ))}
+</ToggleButtonGroup>
+{chartType === 'bar' && <BarChartPreview />}
+{chartType === 'line' && <LineChartPreview />}
+{chartType === 'scatter' && <ScatterChartPreview />}
```

Comment:
This preview is useless

Reply:
Do you know how to prevent the preview from being generated?